### PR TITLE
Add analytics to CI doc build

### PR DESCRIPTION
**Pull request by jakebolewski**
### Description

Add analytics to published doc build to track engagement with project documentation. 

cc @valeriabarra 

- [ ] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [ ] Unit tests are included OR N/A.
- [ ] Code is exercised in an integration test OR N/A.
- [ ] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Comment by valeriabarra** [comment]
Awesome, thanks Jake! 
You can show me later (even privately) how to check/monitors the traffic. 
Thanks!

**Review by valeriabarra — APPROVED** [review]

File: docs/make.jl [hunk 1]
```diff
@@ -46,9 +46,10 @@ mathengine = MathJax(Dict(
 ))
 
 format = Documenter.HTML(
-    prettyurls = get(ENV, "CI", "") == "true",
+    prettyurls = get(ENV, "CI", "") != "" ? true : false,
```
Comment (charleskawczynski) [inline docs/make.jl:49]:
```suggestion
    prettyurls = !isempty(get(ENV, "CI", "")),
```
Maybe a bit simpler to parse?

**Review by charleskawczynski — APPROVED** [review]
I'm not familiar with the analytics keyword, but this looks fine to me.

**Comment by jakebolewski** [comment]
Seems to be working:

image

File: .github/workflows/JuliaFormatter.yml [hunk 2]
```diff
@@ -35,7 +31,7 @@ jobs:
     - uses: julia-actions/setup-julia@latest
       if: steps.filter.outputs.julia_file_change == 'true'
       with:
-        version: 1.5.2
+        version: 1.5.3
```
Comment (valeriabarra) [inline .github/workflows/JuliaFormatter.yml:34]: Sorry I have a few questions. Is this version change needed to fix the current formatter issue? Should we communicate it? Should people also use Julia 1.5.3 locally?

Comment (jakebolewski) [reply .github/workflows/JuliaFormatter.yml:34]: no

Comment (valeriabarra) [reply .github/workflows/JuliaFormatter.yml:34]: ah! Thanks for clarifying

**Pull request merged**